modular_zubbers/code/modules/quirks/positive_quirks/telepathy.dm

```diff
@@ -90,6 +90,10 @@
 
 	last_target_ref = WEAKREF(target)
 
+	var/subtler = FALSE
+	if (message && length(message) > 0 && copytext(message, 1, 2) == "#")
```

Comment:
`message && length(message) > 0` is redundant here because it was already checked in `before_cast`